### ISSUE TITLE
instanceof(global) is undefined rather than "global" (as on native)

### ISSUE THREAD

**Issue by YellowAfterlife** [YoYoGames/GameMaker-HTML5#66]
Snippet:
```js
// Script assets have changed for v2.3.0 see
// https://help.yoyogames.com/hc/en-us/articles/360005277377 for more information
show_debug_message("instanceof={0} typeof={1}", instanceof(self), typeof(self));
```

Windows:
> instanceof=global typeof=struct

HTML5:
> instanceof=undefined typeof=struct

Adding
```js
this.__type = "global";
```
to `yyGameGlobals` constructor works.

**Comment by rwkay** [comment]
if you give us a pull request I will accept it...

**Comment by yoyo-danielc** [comment]
Fixed by #71.
